src/main/java/hivemall/mix/server/MixServer.java

```diff
@@ -171,9 +176,4 @@ private void acceptConnections(@Nonnull MixServerInitializer initializer, int po
             bossGroup.shutdownGracefully();
         }
     }
-
-    public enum ServerState {
```

Comment:
I agree that this pr actually does not need this change.
However, code-style should have  consistent formats; enum location is inconsistent with `MixMessasge` https://github.com/myui/hivemall/blob/master/src/main/java/hivemall/mix/MixMessage.java#L68